utils/protobuf-schema-utilities/src/test/proto/TestOrderingSyntax3.proto

```diff
@@ -14,4 +14,6 @@ message Address {
 
 message Customer {
   string name = 1;
+
+  reserved 2, 15, 10 to 11;
```

Comment:
Same as above.

Reply:
Included in the extension range: https://github.com/Apicurio/apicurio-registry/pull/1727